ironfish/src/sdk.ts

```diff
@@ -133,14 +146,37 @@ export class IronfishSdk {
     }
 
     let client: RpcSocketClient
+    let rpcAuthToken = internal.get('rpcAuthToken')
```

Comment:
I'm worried that automatically generating the token for the client could lead to some confusion.

If I pull down these changes on a remote node, then it will automatically generate a token for the adapter, but when I try to connect with a local client, then it will automatically generate a different mismatched token.

Reply:
If the server and client are not on the same machine, it wont match, which is expected behavior. We want to stop unknown client connect to server. 
The user need to copy paste the internal.json file from server to their local client to pass authentication

Reply:
> If the server and client are not on the same machine, it wont match, which is expected behavior. We want to stop unknown client connect to server. The user need to copy paste the internal.json file from server to their local client to pass authentication

That's true, but generating a token in the client won't match any server. It will only match if a user later starts a server on the same machine

Reply:
ok, I removed the token generation logic since now the token is optional.